### PR TITLE
[DNVB] Footer

### PR DESCRIPTION
**Pull request by flair-conor**
Added footer stylings

#24

### CODE REVIEW

File: templates/theme-components/common/footer.html [hunk 1]
```diff
@@ -42,64 +36,40 @@ <h5 class="footer-info-heading">{{lang 'footer.navigate'}}</h5>
...
-            {{/if}}
+                {{#if settings.address}}
+                    <address>{{nl2br settings.address}}</address>
 
-            <article class="footer-info-col" data-section-type="newsletterSubscription">
-                {{#if settings.show_newsletter_box}}
-                    {{> theme-components/common/subscription-form}}
+                    {{#if settings.phone_number}}
+                        <strong>{{lang 'footer.call_us' phone_number=settings.phone_number}}</strong>
+                    {{/if}}
```
Comment (flair-duncan) [inline templates/theme-components/common/footer.html]: Needs to be outside of `if address` check

File: templates/theme-components/common/footer.html [hunk 2]
```diff
@@ -42,64 +36,40 @@ <h5 class="footer-info-heading">{{lang 'footer.navigate'}}</h5>
...
-        {{/if}}
-        {{#if theme_settings.show_copyright_footer}}
+        {{#or theme_settings.show_copyright_footer theme_settings.show_powered_by}}
             <div class="footer-copyright">
-                <p class="powered-by">&copy; {{moment format="YYYY"}} {{settings.store_name}} </p>
+                {{#if theme_settings.show_copyright_footer}}
+                    <p class="powered-by">Copyright &copy; {{moment format="YYYY"}}&#0044; {{settings.store_name}} </p>
+                {{/if}}
+                {{#and theme_settings.show_copyright_footer theme_settings.show_powered_by}}
+                    <p class="powered-by">&#0046;&nbsp</p>
```
Comment (flair-duncan) [inline templates/theme-components/common/footer.html]: Don't use an empty element please, control this with CSS

File: assets/scss/addons/_settings.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.button {
+    &--primary {
+        font-weight: 600;
```
Comment (flair-duncan) [inline assets/scss/addons/_settings.scss]: should be using a preset

File: config.json [hunk 4]
```diff
@@ -296,7 +296,7 @@
     "restrict_to_login": false,
     "swatch_option_size": "22x22",
     "social_icon_placement_top": false,
-    "social_icon_placement_bottom": "bottom_none",
+    "social_icon_placement_bottom": "bottom_right",
```
Comment (flair-duncan) [inline config.json]: please use a more fitting description - in the theme it's not showing bottom right. Change to a bool value?

**Comment by flair-duncan** [comment]
- Vertical spacing is also off between elements, compared to footer design. Everything is currently too tight. 
- Subscribe button is too wide.
- Remove italic styling from address

**Comment by flair-duncan** [comment]
@flair-conor please rebase

**Comment by flair-conor** [comment]
@flair-duncan Done

File: schema.json [hunk 5]
```diff
@@ -806,6 +784,40 @@
...
+          {
+            "value": "false",
+            "label": "Off"
+          },
+          {
+            "value": "true",
+            "label": "On"
+          }
+        ]
+      },
```
Comment (flair-duncan) [inline schema.json]: No need to use selects for bool options - please change to checkboxes

**Comment by flair-duncan** [comment]
@flair-conor You'll need to hard reset again and pull before you make your changes

File: schema.json (hunk 5 above)
Comment (flair-conor) [reply schema.json]: Done

**Pull request merged**